prudbg.h

```diff
@@ -10,14 +10,15 @@
 #define PRUDBG_H
 
 // default processor to use if none is specified on the command line when prudebug is started
-#define DEFAULT_PROCESSOR_INDEX	AM335x
+#define DEFAULT_PROCESSOR_INDEX	AM57x1
```

Comment:
I don't like the change of default. I'd leave the old one. Later we could add an autodetect feature